src/winter_onboarding_2021/fleet_management_service/views/fleet.clj

```diff
@@ -7,3 +9,30 @@
     [:div [:label {:for "name" :class "form-label"} "Fleet Name"]
      [:input {:class "form-control" :type "text" :name "name" :id "name" :value ""}]]
     [:button {:type "submit" :class "btn btn-primary"} "Submit"]]])
+
+(def headers
+  [{:label "Name" :value :fleets/name}
+   {:label "Managers" :value :managers}
+   {:label "Created At" :value :fleets/created-at}])
+
+(defn managers-list [managers]
+  (string/join ", " (map #(:users/name %) managers)))
+
+(defn fleet-row [row]
```

Comment:
Lot of instances in this and the next function where destructuring can be employed to make the hiccup more readable.